### ISSUE TITLE
APSIM 7.10 (r4158) / Cotton model can't read updated cultivar parameters

### ISSUE THREAD

**Issue by RomainL01SON** [APSIMInitiative/APSIMClassic#1768]
Dear APSIM community,
I'm trying to modify cotton cultivar parameters (calibration).
In the "Cotton.XML" file, I modified some parameters for the cultivar in use in the .apsim file. 
I saved the XML file.

When I run the simulation, it seems that the values of parameters are not read from that file... The SUM file indicate that the model keeps using the previous values.

Do you have any idea why?

FYI: I have done that for other crops (SUGAR, POTATO etc) and it worked perfectly whether I added a new cultivar or I modified the parameters of an existing one.

**Comment by peter-devoil** [comment]
Yes, cotton has a different mechanism than the rest. I seem to remember having to edit the apsim file (by hand in a text editor) - changing the xml of the cotton2 component.

**Comment by RomainL01SON** [comment]
Thank you very Peter. 
I've modified the parameter values in the .apsim file.
It seems that Cotton.XML is not read although APSIM return error if Cotton.XML is missing from the "model" folder where APSIM has been installed.
I've also removed the cultivar in use (and its parameters) the Cotton.XML file and as long as it is in the .apsim file, it runs perfectly.
I will be able to calibrate my cotton cultivars now.
Thanks a lot!
Cheers
Romain
